ci-operator/config/openshift-priv/cloud-event-proxy/openshift-priv-cloud-event-proxy-release-4.12.yaml

```diff
@@ -71,7 +71,7 @@ tests:
         fi
         popd
         export CNE_IMG=${IMAGE_CLOUD_EVENT_PROXY}
-        export CONSUMER_IMG=quay.io/openshift/origin-cloud-event-proxy:4.18
+        export CONSUMER_IMG=quay.io/redhat-cne/cloud-event-consumer:4.18
```

Comment:
Why consumer vs proxy here?

Reply:
the cloud-event-consumer is the correct CONSUMER_IMG. Previous PR put a wrong image name origin-cloud-event-proxy by mistake.